tests/test_messages.py

```diff
@@ -168,14 +140,6 @@ def test_validate_get_composite_profile_payload():
     validate_payload(message, ocpp_version="1.6")
 
 
-def test_get_schema_with_invalid_name():
```

Comment:
do we have a test_get_validator_with_invalid_name ?

Reply:
Nope but there is now.